docs/csaf_checker.md

```diff
@@ -26,6 +26,11 @@ Help Options:
   -h, --help                     Show this help message
 ```
 
+The checker will attempt to download from the given domain URL. 
```

Comment:
A domain is not an URL, both ways should be documented separately. And probably only once as there are three places where it can be used: checker, downloader and aggregator